test/integration/default/controls/service.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+databases = input('pillar').dig('collectd', 'plugins', 'postgresql', 'databases')
```

Comment:
I don't think those `input` directives work, no input is declared in the `kitchen.yml`. Which makes the following tests (`.each` loops) not run. You can see here https://gitlab.com/saltstack-formulas/collectd-formula/-/jobs/2396061477 that none of those tests are executed.